services/payload/src/components/SideNav/SideNav.scss

```diff
@@ -0,0 +1,78 @@
+.navbar-wrapper {
+    background-color: #FFFFFF;
+    box-shadow: 4px 0px 4px 0px rgba(0, 0, 0, 0.10);
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
```

Comment:
`100vh` is kind of weird and IMO shouldn't get used because of the URL bar appearing/disappearing on mobile. Instead, when using position fixed like this, `100%` is _usually_ better! Or `dvh` (see [this guy](https://css-tricks.com/the-large-small-and-dynamic-viewports/) for like a small thing about it)